examples/web-ui/material/menus/menu.smash

```diff
@@ -0,0 +1,18 @@
+* Navigate to first level page {{page}}
+    setFirstLevelMenuItems +? {
+        props({
+            'hamburger': `body > header > button.header__hamburger.header__button`,
+            'layoutButton': `body > nav.side-nav.initialized > ul:nth-child(5) > li:nth-child(3) > button`
+        })
```

Comment:
Props should only be defined in an `* On [page]` function. I'm going to push this pattern really hard, since I feel there should be a standard.